image/cli/mascli/functions/gitops_db2u_database

```diff
@@ -569,6 +569,11 @@ DB2_WORKLOAD: '${DB2_WORKLOAD}'"
     echo_h2 "Commit and push changes to GitHub repo $GITHUB_ORG $GITHUB_REPO"
     save_to_target_git_repo $GITHUB_HOST $GITHUB_ORG $GITHUB_REPO $GIT_BRANCH $GITOPS_APPS_DIR "${GIT_COMMIT_MSG}"
 
+    echo "git: Added ${FILES_ADDED} files"
+    if [ "$FILES_ADDED" != "0" ]; then
```

Comment:
this section needs to be after the sync and health check as the db2u pod won't be available until the db2u operator is processed that commits